change/@ni-nimble-components-b13941ed-a21b-4658-8b5a-7cb96ea812f1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@vivinkrishna-ni I'm not sure if you saw it but @NIbokeefe filed #2351 for another calendar-related token that's missing. Will you be the one to add that, either in this PR or a follow up?

Reply:
Thanks for letting us know. Yes, we will add the new color token in a follow-up PR. I have created a task in AzDo to track it: https://dev.azure.com/ni/DevCentral/_workitems/edit/2835500